tests/general_tests.py

```diff
@@ -89,6 +90,21 @@ async def func_headers_buffer_size():
         async with session.get(url, headers=headers_buffer_size) as response:
             await print_message(response.status, 200, "headers > buffer_size")
 
+async def file_sender(file_name):
+	try:
+		async with aiofiles.open(file_name, 'rb') as f:
+			chunk = await f.read(900)  # 900 bytes chunks
+			while chunk:
+				yield chunk
+				chunk = await f.read(1024)
+		print("Finished reading file.")
+	except Exception as e:
+		print(f"Error during file reading: {e}")
+
+async def send_chunked_request(file_name):
+	async with aiohttp.ClientSession() as session:
+		async with session.post(url, data=file_sender(file_name)) as response:
+			 await print_message(response.status, 200, "chunked request with file")
 
```

Comment:
Did you get what was the problem we had before and how we solved it?

Reply:
First of all, we did not return true from `Connection::readChunkedBody`.

2. We did not allow this type to pass `type == "application/octet-stream"`

3. POST logic on `readFromClient` was slightly incorrect. I had to add `else if (!conn.getHasReadSocket())` to the readBody

4. Python code was completely incorrect!